qgsquick/from_qgis/plugin/editor/qgsquickeditorwidgetcombobox.qml

```diff
@@ -78,4 +79,26 @@ ComboBox {
     }
   }
   // [/hidpi fixes]
+
+  indicator: Item {
+    anchors.fill: parent
+
+    Image {
+      id: comboboxIndicatorIcon
+      source: QgsQuick.Utils.getThemeIcon("ic_angle_double_down")
+      height: parent.height * 0.4
```

Comment:
Would be good to use unified size icon. Could you please check how it is for other widgets with icon?